src/makeCalculator.js

```diff
@@ -4,7 +4,46 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(a) {
+      calculator.result += a;
```

Comment:
you can work with `this`
```suggestion
      this.result += a;
```